packages/arb-avm-cpp/avm/src/machine.cpp

```diff
@@ -77,11 +78,14 @@ Assertion Machine::run() {
     uint256_t initialConsumed = machine_state.getTotalMessagesRead();
     uint32_t delayAbortCheckCounter = 0;
     while (true) {
-        if (delayAbortCheckCounter >= 100) {
+        if (delayAbortCheckCounter >= 1000000) {
             if (is_aborted.load(std::memory_order_relaxed)) {
                 break;
             }
             delayAbortCheckCounter = 0;
+
+            // Allow other threads to run
+            std::this_thread::sleep_for(std::chrono::seconds(0));
```

Comment:
I believe the standard way to do this is `std::this_thread::yield();`

Reply:
:+1: